chromium_src/chrome/browser/ui/views/tabs/tab_strip_layout_helper.cc

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/tabs/brave_tab_strip_layout.h"
+#include "chrome/browser/ui/views/tabs/tab_strip_layout.h"
+
+#define CalculateTabBounds                                                    \
```

Comment:
who use this?

Reply:
https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/views/tabs/tab_strip_layout_helper.cc;l=275;drc=991d3fc291bb5ab890f9d6cd0ee0c230be3edb01

It's used in tab_strip_layout_helper.cc and it's the only occurrence in that file.

Reply:
ah, I see 👍🏼